examples/where+stdev/where+stdev.ipynb

```diff
@@ -165,93 +136,72 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 82,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "tensor(266., dtype=torch.float64)\n",
-      "tensor(True)\n",
-      "bool:  tensor(True)\n"
+      "/Users/mhchia/projects/work/pse/zk-stats-lib/zkstats/computation.py:166: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.\n",
+      "  is_precise_aggregated = torch.tensor(1.0)\n",
+      "/Users/mhchia/Library/Caches/pypoetry/virtualenvs/zkstats-brXmXluj-py3.12/lib/python3.12/site-packages/torch/onnx/symbolic_opset9.py:2174: FutureWarning: 'torch.onnx.symbolic_opset9._cast_Bool' is deprecated in version 2.0 and will be removed in the future. Please Avoid using this function and create a Cast node instead.\n",
+      "  return fn(g, to_cast_func(g, input, False), to_cast_func(g, other, False))\n",
+      "/Users/mhchia/Library/Caches/pypoetry/virtualenvs/zkstats-brXmXluj-py3.12/lib/python3.12/site-packages/torch/onnx/utils.py:1703: UserWarning: The exported ONNX model failed ONNX shape inference. The model will not be executable by the ONNX Runtime. If this is unintended and you believe there is a bug, please report an issue at https://github.com/pytorch/pytorch/issues. Error reported by strict ONNX shape inference: [ShapeInferenceError] (op_type:Where, node name: /Where): Y has inconsistent type tensor(double) (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/torch/csrc/jit/serialization/export.cpp:1490.)\n",
+      "  _C._check_onnx_proto(proto)\n"
      ]
     }
    ],
    "source": [
-    "# Verifier/ data consumer side:\n",
-    "class verifier_model(nn.Module):\n",
-    "    def __init__(self):\n",
-    "        super(verifier_model, self).__init__()\n",
-    "        # w represents mean in this case\n",
-    "        self.w = nn.Parameter(data = dummy_theory_output, requires_grad = False)\n",
-    "        self.data_mean = nn.Parameter(data = dummy_data_mean, requires_grad = False)\n",
+    "# Verifier/ data consumer side: send desired calculation\n",
+    "from zkstats.computation import computation_to_model, State\n",
+    "\n",
+    "\n",
+    "def computation(s: State, data: list[torch.Tensor]) -> torch.Tensor:\n",
+    "    x = data[0]\n",
+    "    # FIXME: should be replaced by `s.where` when it's available. Now the result may be incorrect\n",
+    "    filter = (x > 5)\n",
+    "    filtered_x = torch.where(filter, x, 0.0)\n",
```

Comment:
We need other filters like `fil_mean_X` and `fil_std_X` to make it run correctly. Or probably we can fix it later when `State.where` is supported?

Reply:
Yes, or maybe we can encapsulate the logic in s.mean to detect if it is addressing the filter data or not. Will open a new PR for this